CHANGELOG

```diff
@@ -3,6 +3,7 @@ Please view this file on the master branch, on stable branches it's out of date.
 v 7.14.0 (unreleased)
   - Disable turbolinks when linking to Bitbucket import status (Stan Hu)
   - Fix broken code import and display error messages if something went wrong with creating project (Stan Hu)
+  - Allow displaying of archived projects in the admin interface (Artem Sidorenko)
```

Comment:
@Razer6 in the Changelog in master is no `v 8.0`. Should I create it and move it?